### PR TITLE
FISH-654 Payara Micro tools in Eclipse IDE

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/payara-micro.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
+= Payara Micro tools in Eclipse IDE
+
+[[create-micro-maven-project]]
+== Creating new Payara Micro Maven Application
+
+* Simply start up Eclipse IDE and then Choose *File > New > Project* from the main menu
+ or press the Ctrl+N to get the list of available projects:
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/payara-micro.adoc:7]:
```suggestion
 or press Ctrl+N to get the list of available projects:
```

File: docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/payara-micro.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+image::eclipse-plugin/payara-micro/micro-settings.png[Micro settings]
+
+* The IDE creates the Payara Micro application using the Payara Micro maven archetype. 
+As you can see, the project has a basic structure with *src\main\java* and *src\main\webapp* folders:
+
+image::eclipse-plugin/payara-micro/project-pom.png[Micro project generated]
+
+[[run-micro-application]]
+== Running the Payara Micro Application
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/payara-micro.adoc:35]:
```suggestion
== Running The Payara Micro Application
```

File: docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/payara-micro.adoc [hunk 3]
```diff
@@ -0,0 +1,77 @@
+= Payara Micro tools in Eclipse IDE
+
+[[create-micro-maven-project]]
+== Creating new Payara Micro Maven Application
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/payara-micro.adoc:4]:
```suggestion
== Creating A New Payara Micro Maven Application
```

File: docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/README.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
+= Payara Eclipse IDE Plugin
+
+Payara Platform provides the Payara Tools plugin to integrate Payara Server and Payara Micro into Eclipse IDE.
+The plugins are installable from the https://marketplace.eclipse.org/content/payara-tools[Eclipse Marketplace].
+
+[[installing]]
+== Installing Payara Tools plugin
```
Comment (AlanRoth) [inline docs/modules/ROOT/pages/documentation/ecosystem/eclipse-plugin/README.adoc:7]:
```suggestion
== Installing Payara Tools Plugin
```

**Review by AlanRoth — APPROVED** [review]
Just some nitpicks, otherwise LGTM

**Comment by jGauravGupta** [comment]
Thanks @AlanRoth

**Pull request merged**